blogposts/2020/3.18/announcing-sourcegraph-3.18.md

```diff
@@ -0,0 +1,74 @@
+
+-   [Small but mighty new features](https://about.sourcegraph.com/blog/small-and-mighty-features)
+
+    Sourcegraph 3.18 includes several small features that have a big impact. Based on lots of feedback, we added support for campaigns in GitLab. We’ve also improved privacy feedback when adding repositories, and created custom search pages for Sourcegraph Cloud.
+
+-   [New C++ precise code intelligence solution](https://about.sourcegraph.com/blog/c-plus-plus-code-intel)
+
+    We wrote a new LSIF indexer based on clangd. This means that developers using C and C++ now have access to precise code intelligence. Developers using the original C++ LSIF indexer based on DXP will see noticeable improvements with this new implementation; faster than before, and now with hovers.
+
+-   [Getting notified about the health of Sourcegraph is even easier](https://about.sourcegraph.com/blog/sourcegraph-health-notification)
+
+    Alerts are now delivered with detailed information, including links to potential solutions. Additionally, we now monitor container health and over/under provisioning in all deployment modes -- so you will even get alerts if Sourcegraph needs more resources or could do with less!
+
+-   [Custom search pages on Sourcegraph Cloud](https://about.sourcegraph.com/blog/custom-search-pages)
+
+    Sourcegraph 3.18 has added new search pages that scope searches to a collection of repositories. This helps new users coming to Sourcegraph search over code that is more relevant to them, and introduces them to interesting queries in the process. Many communities will be excited to pick this up and have pages created specifically for them (for example, Kubernetes).
+
+-   [Changelog](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@master/-/blob/CHANGELOG.md)
+
+    All the details for what changed and why, can be found here.
+
+
+## Thank you!
+
+Thank you to the many people who contributed to Sourcegraph since the last release!
+
+-   [@Jaxkr](https://github.com/Jaxkr)
+-   [@aisbaa](https://github.com/aisbaa)
+-   [@abitrolly](https://github.com/abitrolly)
+-   [@smaifullerton-wk](https://github.com/smaifullerton-wk)
+-   [@terinjokes](https://github.com/terinjokes)
+-   [@svrx](https://github.com/svrx)
+-   [@DolceTriade](https://github.com/DolceTriade)
+-   [@shadyabhi](https://github.com/shadyabhi)
+-   [@blanet](https://github.com/blanet)
+-   [@bb-qq](https://github.com/bb-qq)
+-   [@byrongrogan](https://github.com/byrongrogan)
+-   [@mipearson](https://github.com/mipearson)
+-   [@kwangil-ha](https://github.com/kwangil-ha)
+-   [@s-yadav](https://github.com/s-yadav)
+-   [@lf-](https://github.com/lf-)
+-   [@smcallis](https://github.com/smcallis)
+-   [@zgrimshell](https://github.com/zgrimshell)
+-   [@jlangston](https://github.com/jlangston)
+
+## Share your feedback
+
+We want to hear what you think about Sourcegraph! If there’s something coming you are curious about or you have a specific feature you want to request,
```

Comment:
This is an area where we could sprinkle in some social proof. "Whether you use Sourcegraph 20 times per day or are new to it, we'd love to hear what you think: (bullet list)"

Reply:
done